packages/eslint-config/index.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
```

Comment:
We need to revisit this whole config. 
This is a typescript parser which is something we are not supporting.

Reply:
@ankurkakroo2  done